packages/editor/src/editor-styles/ast/parse.js

```diff
@@ -0,0 +1,685 @@
+// Adapted from https://github.com/reworkcss/css
+// because we needed to remove source map support.
```

Comment:
did we consider creating an issue or PR on the upstream project to allow things like disabling source maps? it appears like there are a few related issues concerning this…

even if we created an actual fork of the project then included the fork here it could stand a better chance of merging upstream to the point where we could remove the fork later, no?

Reply:
Yes, good call. I forgot to do that. I'll create an issue upstream, not certain how can they support this but yeah I'll open an issue and see how to move forward depending on the feedback.